packages/react-components/src/organisms/RichTextCard.tsx

```diff
@@ -1,17 +1,25 @@
-import React from 'react';
+import React, { Fragment } from 'react';
 import { Card, Headline2 } from '../atoms';
 import RichText from './RichText';
+import { Collapsible } from '../molecules';
 
-interface RichTextCardProps {
+type RichTextCardProps = {
   readonly title: string;
   readonly text?: string | null;
-}
-const RichTextCard: React.FC<RichTextCardProps> = ({ title, text }) =>
-  text ? (
+  readonly collapsible?: boolean;
+};
+const RichTextCard: React.FC<RichTextCardProps> = ({
+  title,
+  text,
+  collapsible = false,
+}) => {
+  const wrapper = collapsible ? Collapsible : Fragment;
+  return text ? (
     <Card>
       <Headline2 styleAsHeading={3}>{title}</Headline2>
-      <RichText text={text} />
+      {wrapper({ children: <RichText text={text} /> })}
```

Comment:
You wanna do something like `<Wrapper><RichText text={text} /></Wrapper>` (wrapper variable needs to be uppercase for it to work. This literally calls the component function which is dangerous